### PR TITLE
Restore local sphinx build

### PR DESCRIPTION
**Pull request by bast**
removed extensions why are not needed at this moment

### CODE REVIEW

**Comment by ilfreddy** [comment]
Thanks Radovan! 
conf.py from @robertodr had some problems.

**Comment by bast** [comment]
(we were also practicing a PR cycle, sorry for spam Roberto)

File: docs/conf.py [hunk 1]
```diff
@@ -41,16 +40,9 @@
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 # source_suffix = ['.rst', '.md']
-# Markdown support
-
-from recommonmark.parser import CommonMarkParser
```
Comment (robertodr) [inline docs/conf.py:46]: @bast @ilfreddy Why remove the possibility to write in Markdown? ReadTheDocs can handle it, given the appropriate `reqruirements.txt`. For a local build one could get the packages rather easily. Either globally:

``` Python
pip install -r docs/requirements.txt
```

or (even better) locally in a [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/) using the same command.

Comment (bast) [inline docs/conf.py:46]: Good point. I did not know about this file.

**Comment by robertodr** [comment]
@bast No problem. And you also got a code review example :wink:

**Pull request merged**